packages/network-query/queries/network/deployments.gql

```diff
@@ -21,6 +21,27 @@ fragment DeploymentIndexerFields on DeploymentIndexer {
   }
 }
 
+fragment DeploymentIndexerNodeFields on DeploymentIndexer {
```

Comment:
Reuse the `IndexerDeploymentFields` instead duplicate the fields, recommend to have a look on https://www.apollographql.com/docs/react/data/fragments/

```
fragment DeploymentIndexerNodeFields on DeploymentIndexer {
  ...DeploymentIndexerFields
  deployment {
    id
    project {
      id
      metadata
    }
  }
}
```

Reply:
Don't include any fragments unless they are being used in a query inside network-query package